src/Accounts/AccountsForm.js

```diff
@@ -47,19 +49,19 @@ class AccountsForm extends Component {
           <Col xs={12} md={6}>
             <Row>
               <Col xs={12}>
-                <Field label="name*" name={`${elem}.name`} id={`${elem}.name`} validate={[Required]} component={TextField} fullWidth />
+                <Field label={<FormattedMessage id="ui-vendors.accounts.nameAst" />} name={`${elem}.name`} id={`${elem}.name`} validate={[Required]} component={TextField} fullWidth />
```

Comment:
Just add attribute `required` to `Field` and it'll show a red asterisk

Reply:
+1, no need to use separate labels, also the same UX within FOLIO platform